src/frontend-samples/missclassification-sample/sampleSpec.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { SampleIModels } from "@itwinjs-sandbox/SampleIModels";
+import { SampleSpec } from "SampleSpec";
+
+export function getMisclassificationSpec(): SampleSpec {
+  return ({
+    name: "misclassification-sample",
+    label: "Misclassification",
+    image: "classification.png",
+    readme: async () => import("!!raw-loader!./readme.md"),
+    iTwinViewerReady: true,
```

Comment:
Can you add a `description` entry? It is what gets indexed for search. Be sure to include some hashtags.  See other sampleSpecs for examples